### PR TITLE
Added Zabbix integration do Zammad documentation

### PR DESCRIPTION
**Pull request by RogerTheUnicornHive**
Hello as requested in https://github.com/zammad/zammad-admin-documentation/pull/50/commits
I'm submitting Zabbix integration to the master branch of Zammad documentation.
Warm Regards,
Alex

### CODE REVIEW

**Comment by MrGeneration** [comment]
Thank you so much for your patience and contribution!

**Comment by RogerTheUnicornHive** [comment]
@MrGeneration thank you :)
I hope you have a prosperous and wonderful end of working week and weekend.

**Comment by MrGeneration** [comment]
Oh you too! :)
I wanted to let you merge so the history keeps your name in a better way. 💪

**Comment by RogerTheUnicornHive** [comment]
It seems that I am unable to merge - seems like permission issue
 Only those with write access to this repository can merge pull requests.

**Comment by MrGeneration** [comment]
Doh! my bad, then I have to do that for you ;)

**Comment by RogerTheUnicornHive** [comment]
No worries. Thanks again for all the assistance.

**Comment by rlue** [comment]
Hey folks! It looks like there might have been a problem with this PR: rather than simply adding to `system/integrations.rst`, it duplicated that file to `system/integrations/system-integrations.rst` and then put the additions there.

The bigger problem here is not just a duplicated file; @RogerTheUnicornHive's changes are **not visible anywhere in the newly published documentation** because the listing of articles on the official docs page links to the old version. You can see this in the console output when compiling the .rst:

```
$ make html
...
checking consistency... system/integrations/system-integrations.rst: WARNING: document isn't included in any toctree
```

I'm about to prepare a new PR to fix this issue. Let me know if I missed anything!

File: system/integrations/system-integrations.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  * Check_MK
+  * Icinga
+  * Monit
+  * Nagios
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+   integrations/zabbix
```
Comment (rlue) [inline system/integrations/system-integrations.rst:44]: Whoops, it also looks like we were missing a newline above this one, which caused the `toctree` directive to fail:

```
system/integrations.rst:41: WARNING: Error in "toctree" directive:
invalid option block.

.. toctree::
   :maxdepth: 1
   :titlesonly:
   integrations/zabbix
 looking for now-outdated files... none found
 pickling environment... done
 checking consistency... /home/rlue/src/zammad-admin-documentation/system/integrations/zabbix.rst: WARNING: document isn't included in any toctree
```

**Comment by MrGeneration** [comment]
Thanks for the work @rlue .
I anticipated the other PRs on admin docs to be through faster (not your fault!) which is why I've been fixing it in them <.<

I've failed the QA on this one and didn't notice the issue until I was working on google channels.

**Pull request merged**